test/common.js

```diff
@@ -445,3 +446,11 @@ exports.fileExists = function(pathname) {
     return false;
   }
 };
+
+exports.busyLoop = function busyLoop(time) {
```

Comment:
If this PR is still up for consideration, I'll update this.  I'm not sure where it stands so I defer to your judgement.